app/src/main/kotlin/compiler/analysis/BuiltinFunctions.kt

```diff
@@ -13,12 +13,29 @@ object BuiltinFunctions {
             Function.Implementation.Foreign("print_int64"),
             false
         ),
+
         "wczytaj" to Function(
             "wczytaj",
             emptyList(),
             Type.Number,
             Function.Implementation.Foreign("read_int64"),
             false
+        ),
+
+        "przedziału" to Function(
```

Comment:
What do you think of adding some documentation there? It's not obvious without reading C code or unit tests whether `max` is inclusive or not and what is the starting value.

Reply:
Added comments to `BuiltinFunctions`. The documentation for this language (including the standard functions) should be in some separate document, anyway.